src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -1296,6 +1306,9 @@ DatabaseShardImp::setFileStats()
         for (auto const& e : shards_)
             if (e.second.shard)
                 shards.push_back(e.second.shard);
+
+        if (shards.empty())
```

Comment:
Why are you duplicating this check which occurs also at line 1303?

Reply:
Done